aspnetcore/grpc/supported-platforms.md

```diff
@@ -12,33 +12,40 @@ uid: grpc/supported-platforms
 
 By [James Newton-King](https://twitter.com/jamesnk)
 
-This article discusses the requirements and supported platforms for using gRPC with .NET.
+This article discusses the requirements and supported platforms for using gRPC with .NET. There are different requirements for the two major gRPC tasks:
 
-gRPC takes advantage of advanced features available in  HTTP/2. HTTP/2 isn't supported everywhere, but a second wire-format using HTTP/1.1 is available for gRPC:
+* Hosting gRPC services on the server.
+* Calling gRPC services from a client.
+
+gRPC takes advantage of advanced features available in HTTP/2. HTTP/2 isn't supported everywhere, but a second wire-format using HTTP/1.1 is available for gRPC:
 
 * [`application/grpc`](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md) - gRPC over HTTP/2 is how gRPC is typically used.
-* [`application/grpc-web`](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-WEB.md) - gRPC-Web modifies the gRPC protocol to be compatible with HTTP/1.1. gRPC-Web can be used in more places, notably it is callable by browser apps. Two advanced gRPC features are no longer supported: client streaming and bidirectional streaming.
+* [`application/grpc-web`](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-WEB.md) - gRPC-Web modifies the gRPC protocol to be compatible with HTTP/1.1. gRPC-Web can be used in more places. gRPC-Web can be used by browser apps and it can be used in networks without complete support for HTTP/2. Two advanced gRPC features are no longer supported: client streaming and bidirectional streaming.
 
 gRPC on .NET supports both wire-formats. gRPC over HTTP/2 is used by default. For information on setting up gRPC-Web, see <xref:grpc/browser>.
```

Comment:
```suggestion
gRPC on .NET supports HTTP 1.1 and HTTP/2 , HTTP/2 is used by default. For information on setting up gRPC-Web, see <xref:grpc/browser>.
```

Reply:
Updated change slightly. Would prefer to talk about wire format than HTTP version.

Reply:
That's fine, only because you define it above.

> HTTP/2 isn't supported everywhere, but a second wire-format using HTTP/1.1 is available for gRPC:

It's common jargon that can be difficult for non-experts. I doubt it MTs.